### DISCUSSION TITLE
Problem: GitHub Actions seems to not pick up CLI options

### DISCUSSION THREAD

**Discussion by eviau-artefactual** [community/community#110331] [Actions] [answered]
### Select Topic Area

Question

### Body

Hello :sparkles: 

I am working on setting up [sphinxcontrib-spelling ](https://sphinxcontrib-spelling.readthedocs.io) on a sphinx workflow.

In the Sphinx `conf.py`, I can set the locale to `en_CA`, and set up a test to use that locale when using `sphinx- build -b spelling`. It works locally, on my computer.

When I try to make a PR to add the spelling tests as CI tests to the documentation project I am working on, it seems to be that the locale isn't picked up. In the logs, I can see spelling errors that are locale specific. I tried using the `-D` flag to set up the locale directly in the call, but this also doesn't work. Here is the Sphinx doc on that flag: https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-D

Trying to set up the `-D` flag locally seems to work: it will or will not flag locale specific spelling differences accordingly.

So my current best bet is that github actions aren't picking up the options of `sphinx-build`... but maybe I am wrong.

Would you be able to provide me with some hints or things to look at to try and debug this?

I was unable to phrase this request in a way that would allow me to find similar discussions - if you have links to one of them, please share ! :) 

Thank you for your time & attention

**Comment by Lanrad23** [comment, marked as answer]
As far as I know, the CI environment may differ from your local setup, especially in terms of installed locales. You might need to explicitly set up the required locale in the GitHub Actions workflow.

**Reply by eviau-artefactual** [reply]
Yes it was exactly that ! Thank you!

The solution: `sudo apt-get install -y hunspell-en-ca hyphen-en-ca` in the `run` section of the github action workflow.

There are a variety of ways to obtain locales I think... I looked up how sphinxcontrib-spelling infers locales, and it is from PyEnchant, which gets them from hunspell and nuspell - [from their docs](https://pyenchant.github.io/pyenchant/faq.html#how-can-i-use-a-custom-location-for-storing-enchant-dictionaries)

Thanks again :sparkles:

**Comment by Lanrad23** [comment]
Any time, my friend :)
